core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -801,9 +820,14 @@ public void run()
       System.exit(0);
     }
 
+    final Tracer tracer = new Tracer.Builder("YCSB " + workload.getClass().getSimpleName()).conf(conf).build();
```

Comment:
style: these builder chains are easier to read when operations are broken one per line
e.g.
```
Tracer.Builder("YCSB " + workload.getClass().getSimpleName())
    .conf(conf)
    .build();
```